commands/command_edit.cpp

```diff
@@ -910,10 +910,16 @@ void PasteDives::redo()
 	undo();
 }
 
-// ***** Paste *****
+// ***** ReplanDive *****
 ReplanDive::ReplanDive(dive *source, bool edit_profile) : d(current_dive),
+	when(0),
```

Comment:
Not sure about these - I think @atdotde already made these and I objected.

If `d` is NULL, `workToBeDone` returns false and the object will be destroyed immediately. Of course, Coverty isn't that smart.

Up to you.

Reply:
I looked at this and went back and forth. It's silly to initialize things.
I could list it as a false positive and ignore it in the Coverity output. In a way, this is the lazy way out...